### DISCUSSION TITLE
Diary-又京了（三） | Zi-Zi's Journey

### DISCUSSION THREAD

**Discussion by giscus** [GZ-Metal-Cell/GZ-Metal-Cell.github.io#336] [Announcements]
# Diary-又京了（三） | Zi-Zi's Journey

这辈子又又又可以见到小迷糊了！

https://gz-metal-cell.github.io/2024/06/18/Diary-%E5%8F%88%E4%BA%AC%E4%BA%86%EF%BC%88%E4%B8%89%EF%BC%89/

**Comment by promefire** [comment]
Metal哥的体力就是这么练出来的😭太强了😭

**Reply by GZ-Metal-Cell** [reply]
😭游山玩水不务正业😭不要向我学习😭
